usr/share/lib/img_proof/tests/SLES/test_sles_hardened.py

```diff
@@ -0,0 +1,34 @@
+import pytest
+
+
+def test_sles_hardened(host, get_release_value, is_sles_sap, swapfile):
```

Comment:
swapfile still isn't invoked here so setup wouldn't happen. Since this is down to one test module I think it's fine to pull the code from fixture back into the test. There's no longer code duplication. The simpler solution if we want to handle unexpected exceptions would be try/except/finally. Wrap everything in try, raise pytest.fail with exception info, and have the cleanup in finally.